obelisk_ws/src/cpp/obelisk_leap_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.8)
+project(obelisk_leap_cpp)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(dynamixel_sdk REQUIRED)
+
+# Get the Obelisk library
+include(FetchContent)
+FetchContent_Declare(
+  Obelisk
+  SOURCE_DIR $ENV{OBELISK_ROOT}/obelisk/cpp
+)
+
+# Lets us Get Obelisk::Core and Obelisk::Zoo
+FetchContent_MakeAvailable(Obelisk)
+
+include_directories("$ENV{OBELISK_ROOT}/.pixi/envs/dev/include/dynamixel_sdk")
+include_directories("$ENV{OBELISK_ROOT}/.pixi/envs/dev-no-gpu/include/dynamixel_sdk")
```

Comment:
same comment as above. avoid using pixi include dirs here in case end user isn't using pixi to manage deps. we need some other way to specify this.